neuro_san/internals/run_context/langchain/llms/llm_policy.py

```diff
@@ -71,3 +95,33 @@ async def delete_resources(self):
         any web client references in there.
         """
         raise NotImplementedError
+
+    def create_llm_resources_components(self, config: Dict[str, Any]) -> Tuple[BaseLanguageModel, LlmPolicy]:
+        """
+        Basic policy framework method.
+        Most LLMs will not need to override this.
+
+        :param config: The fully specified llm config
+        :return: The components that go into populating an LlmResources instance.
+                This is a tuple of (BaseLanguageModel, LlmPolicy).
+                It's entirely fine if the LlmPolicy is not the same instance as this one.
+        """
```

Comment:
This method is the basic outline of how we generally expect subclass's implementations of create_client() and create_llm() to be called.  If they really know what they are doing, there's nothing stopping an implementation from overriding this, should this be insufficient.